Airports/EGJJ/Sector.txt

```diff
@@ -15,51 +15,249 @@ DISPLAY:JJAPP:JJAPP:JJCTR
 DISPLAY:JJCTR:Only JJAPP:JJCTR
 DISPLAY:JJCTR-S:Only JJAPP:JJCTR
 
-SECTORLINE:JJCTR-S6
-DISPLAY:London S6:JJCTR:London S6
-DISPLAY:JJCTR:JJCTR:London S6
-COORD:N050.00.00.000:W002.40.34.000 ;Top Middle
-COORD:N049.59.60.000:W002.29.35.000
-
-SECTORLINE:JJCTR-LS
-DISPLAY:London AC Worthing:JJCTR:London S6
-DISPLAY:JJCTR:JJCTR:London AC Worthing
-COORD:N049.59.60.000:W002.29.35.000
-COORD:N050.00.00.000:W001.47.00.000
-
-SECTORLINE:JJCTR-LFRR
-DISPLAY:JJCTR:JJCTR:LFRR low
-COORD:N050.00.00.000:W001.47.00.000
-COORD:N049.44.00.000:W002.00.00.000
+;MIDDLEBIT APP
+SECTORLINE:JJAPP-M:
```

Comment:
All of these SECTORLINE: lines have an extra (unnecessary) colon at the end of them